src/utils/EventBus.ts

```diff
@@ -0,0 +1,29 @@
+export class EventBus {
+  private readonly listeners: Record<string, Array<(...args: any[]) => void>> = {};
```

Comment:
Предлагаю детальнее типизировать подписчиков
```suggestion
  private readonly listeners: Record<string, Array<(...args: TListenter[]) => void>> = {};
```
где
```
type TListener = (...args: unknown[]) => void;
```